jest.config.js

```diff
@@ -4,10 +4,18 @@ module.exports = {
   moduleDirectories: ['node_modules', '<rootDir>/src'],
   collectCoverageFrom: [
     '<rootDir>/src/**',
+    '!<rootDir>/src/index.ts',
     '!**/__snapshots__/**',
-    '!**/story-components.tsx',
-    '!**/stories.tsx',
+    '!**/_stories/**',
   ],
+  coverageThreshold: {
```

Comment:
This is pretty gutsy - we can decrease these numbers if we decide 100% coverage isn't reasonable.  But hey, if we're already here, why not aim for it?